app/controllers/draft_vaccination_records_controller.rb

```diff
@@ -57,6 +58,12 @@ def update
 
   private
 
+  def validate_patient_id
+    if current_step == :confirm && @draft_vaccination_record.patient_id.nil?
```

Comment:
I think we could try and come up a more generic approach here. Although we've only seen the case in Sentry where a user navigates to `/draft-vaccination-record/confirm`, there's no reason why they couldn't navigate to `/draft-vaccination-record/batch` without a patient or a session.

Reply:
Thinking about this some more, it's not specific to the `draft-vaccination-record` route either, any of the `draft-` routes could encounter this problem.